bayesian/utils.py

```diff
@@ -349,47 +352,52 @@ def get_analyses_from_graph(ecosystem, package, version):
 
         if graph_req is not None:
             resp = graph_req.json()
+            result_data = resp['result'].get('data')
 
-            if not (resp['result'].get('data') and len(resp['result'].get('data')) > 0):
+            if not (result_data and len(result_data) > 0):
                 # trigger unknown component flow in API for missing package
                 return None
 
             clubbed_data.append({
-                "epv": resp['result']['data']
+                "epv": result_data
             })
 
-            if "cve" in resp['result'].get('data')[0]:
-                script2 = """\
-g.V().has('pecosystem', ecosystem).has('pname', name).has('version', version)\
-.in('has_version').out('has_version').not(out('has_cve')).values('version').dedup();\
-"""
-                payload = {
-                    'gremlin': script2,
-                    'bindings': {
-                        'ecosystem': ecosystem,
-                        'name': package,
-                        'version': version
-                    }
-                }
-
-                graph_req2 = post(gremlin_url, data=json.dumps(payload))
-                if graph_req2 is not None:
-                    resp2 = graph_req2.json()
+            if "cve" in result_data[0]:
```

Comment:
@yzainee is this check enough to conclude that there is a CVE present in the `result.data.cve.cve_id` list?

Reply:
yes, the "cve" will be a part of the json only if the cve exists for that package. The query is written in such a manner